lib/IRGen/IRGenSIL.cpp

```diff
@@ -2260,9 +2260,11 @@ void IRGenSILFunction::visitFloatLiteralInst(swift::FloatLiteralInst *i) {
 static llvm::Constant *getAddrOfString(IRGenModule &IGM, StringRef string,
                                        StringLiteralInst::Encoding encoding) {
   switch (encoding) {
-  case swift::StringLiteralInst::Encoding::UTF8:
-    return IGM.getAddrOfGlobalString(string);
-
+  case swift::StringLiteralInst::Encoding::UTF8: {
+    return IGM.getAddrOfGlobalString(string,
+                                     /*willBeRelativelyAddressed=*/false,
+                                     /*addNull=*/false);
```

Comment:
Not quite yet. Several parts of Swift still rely on `getAddrOfGlobalString` to null-terminate strings.